vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/promotion/get_available/get_available.cc

```diff
@@ -30,14 +30,16 @@ GetAvailable::~GetAvailable() = default;
 
 std::string GetAvailable::GetUrl(const std::string& platform) {
   const auto wallet = ledger_->wallet()->GetWallet();
-  if (!wallet) {
-    BLOG(0, "Wallet is null");
-    return "";
+  std::string payment_id;
+  if (wallet) {
+    payment_id = base::StringPrintf(
+      "&paymentId=%s",
+      wallet->payment_id.c_str());
   }
 
   const std::string& arguments = base::StringPrintf(
-      "migrate=true&paymentId=%s&platform=%s",
-      wallet->payment_id.c_str(),
+      "migrate=true%s&platform=%s",
```

Comment:
Missing a "&" after "true"?

What happens if `payment_id` is empty?

Reply:
not missing `&` as if payment id is "", you get `&` from `platform`, but if `payment_id` exists you get it from `&paymentId=%s`